static/src/stylesheets/module/content-garnett/live-blog/_live-blog.head.scss

```diff
@@ -908,3 +913,43 @@ $pillars: (
         padding-left: $gs-gutter / 2;
     }
 }
+
+// Dead blog overrides
+@each $pillar, $palette in $pillars {
+    .content--liveblog.tonal--tone-dead.content--pillar-#{$pillar} {
+        .tonal__main--tone-dead {
+            @include mq(tablet) {
+                padding-top: $gs-gutter;
+            }
+        }
+
+        .content__head,
+        .content__standfirst {
+            color: $garnett-neutral-1;
+        }
+
+        .content__section-label__link[href] {
```

Comment:
how come we need this `[href]`?

Reply:
It's what's been used by @regiskuckaertz throughout, so need to target the in the same way to override

Reply:
My bad, I added it to the general `a` selector because I had occurrences of links without the `href` attribute. In hindsight, I should rather have added a specific `a:not([href])` rule.